applications/asset_tracker_v2/src/modules/led_module.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <stdio.h>
+#include <logging/log.h>
+#include<toolchain/common.h>
+#include "led_module.h"
+#include <device.h>
+#include <caf/events/led_event.h>
+#include "events/led_state_event.h"
+#include <caf/events/module_state_event.h>
```

Comment:
there are multiple strategies of ordering includes but the one I find neat is:
- system headers
- sdk headers
- local headers

That would look like:
```
#include <stdio.h>
#include <zephyr.h>
#include <caf/event/event.h>
#include "event/event.h"
```

I personally make exception for logging to keep it close to LOG_MODULE_REGISTER.

```
<logging/log.h>
LOG_MODULE_REGISTER(led, CONFIG_LED_CONTROL_LOG_LEVEL);
```

and for module state event as it requires name to be defined first
```
#define MODULE led_state
#include <caf/events/module_state_event.h>
```